### PR TITLE
added bugzilla automation for 1812688

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### Test Result 

```
/home/okhatavk/okhatavk/Satellite/robottelo/env/bin/python /home/okhatavk/Downloads/pycharm-community-2018.1.4/helpers/pycharm/_jb_pytest_runner.py --target test_ldap_authentication.py::test_deleted_idm_user_should_not_be_able_to_login
Launching py.test with arguments test_ldap_authentication.py::test_deleted_idm_user_should_not_be_able_to_login in /home/okhatavk/Satellite/robottelo/tests/foreman/ui

============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.1, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo
plugins: forked-1.1.3, cov-2.8.1, mock-1.10.4, xdist-1.34.0, services-1.3.12020-10-07 07:06:16 - conftest - DEBUG - Collected 1 test cases
collected 1 item


==================== 1 passed, 5 warnings in 344.57 seconds ====================
```

Gitlab PR raised : 354

### CODE REVIEW

File: tests/foreman/ui/test_ldap_authentication.py
Comment (akhil-jha) [inline tests/foreman/ui/test_ldap_authentication.py:1450]: I know this works but just a suggestion, maybe we could use:
`sat_admins, sat_users = ipa_data['groups'].replace(" ", '').split(',')`
`idm_admin, idm_user = ipa_data['group_users'].replace(" ", '').split(',')`

but then again, you'll have to add in satellite_auth.py::ipa_data

Comment (omkarkhatavkar) [reply tests/foreman/ui/test_ldap_authentication.py:1450]: I think it is fine that if we are using the settings as I not needed much data from ipa_data. on other had we we required to access the fixture for no significant advantages.

File: tests/foreman/ui/test_ldap_authentication.py
Comment (akhil-jha) [inline tests/foreman/ui/test_ldap_authentication.py:1482]: Don't we have to use `settings.ipa.password_ipa` instead of `settings.server.ssh_password` ?

File: tests/foreman/ui/test_ldap_authentication.py
Comment (akhil-jha) [inline tests/foreman/ui/test_ldap_authentication.py:1453]: Correct me if I am wrong but I see we are not using ActivationKey permission and it's being used only for limiting the user's activities? 
So if that's the case, just a suggestion, we could just replace it with `user_permissions = {None: ['access_dashboard']}`

File: tests/foreman/ui/test_ldap_authentication.py
Comment (akhil-jha) [inline tests/foreman/ui/test_ldap_authentication.py:1480]: So here we are using ` with Session(user=settings.server.admin_username, password=settings.server.admin_password)` which I guess is equivalent to `with session: `, why are we not using that?

Comment (omkarkhatavkar) [reply tests/foreman/ui/test_ldap_authentication.py:1480]: your assumption is wrong `with session: ` uses the user-created from the https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/conftest.py#L132 we not wanted in this test case

File: tests/foreman/ui/test_ldap_authentication.py
Comment (omkarkhatavkar) [reply tests/foreman/ui/test_ldap_authentication.py:1453]: In this test, we are testing user with admin and non_admin permissions, how above `user_permissions = {None: ['access_dashboard']}` different from what ever currently exist ?

Comment (akhil-jha) [reply tests/foreman/ui/test_ldap_authentication.py:1453]: hmm..I was thinking of some other test. My bad.
Resolving this conversation.

**Comment by akhil-jha** [comment]
Thanks for answering my queries. Looks good to me.

**Comment by omkarkhatavkar** [comment]
> Thanks for answering my queries. Looks good to me.

Please give ACK on review.

**Comment by akhil-jha** [comment]
> Thanks for answering my queries. Looks good to me.

ACK

**Review by akhil-jha — APPROVED** [review]
ACK

File: tests/foreman/ui/test_ldap_authentication.py
Comment (mirekdlugosz) [inline tests/foreman/ui/test_ldap_authentication.py:1437]: Is step 6 correct? It doesn't describe what is actually happening in test.

**Review by mirekdlugosz — APPROVED** [review]
I think I've seen config-related code already in another PR. I take that you guys control that and will be rebasing one PR once another is merged?

Left one non-blocking comment

File: tests/foreman/ui/test_ldap_authentication.py
Comment (omkarkhatavkar) [reply tests/foreman/ui/test_ldap_authentication.py:1437]: I will update that, basically, we wanted to have a user with non-admin permissions

**Comment by omkarkhatavkar** [comment]
> I take that you guy

we both will be taking care of the conflicts in case anyone PR get merged first

**Review by mirekdlugosz — APPROVED** [review]

File: tests/foreman/ui/test_ldap_authentication.py
Comment (lhellebr) [inline tests/foreman/ui/test_ldap_authentication.py:114]: `tear down groups`

File: tests/foreman/ui/test_ldap_authentication.py
Comment (lhellebr) [inline tests/foreman/ui/test_ldap_authentication.py:1482]: I think you only test if the external user is displayed, right? But in the docstring, you say that both external AND internal users should be tested.

**Review by lhellebr — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_ldap_authentication.py
Comment (lhellebr) [inline tests/foreman/ui/test_ldap_authentication.py:1425]: `All the users (internal and external) should be displayed to all LDAP admins (internal and external).`

File: tests/foreman/ui/test_ldap_authentication.py
Comment (omkarkhatavkar) [reply tests/foreman/ui/test_ldap_authentication.py:1482]: You are correct we are only testing external user with internal and external user admin's. I will correct doc-string

**Review by lhellebr — APPROVED** [review]

**Comment by lhellebr** [comment]
LGTM but there's a conflict, can't merge

**Comment by omkarkhatavkar** [comment]
@lhellebr let me resolve it

**Comment by lhellebr** [comment]
Please provide test results after rebase and I will merge once travis is complete

**Comment by omkarkhatavkar** [comment]
### Test Result 
```
(env) /home/okhatavk/Satellite/robottelo (bz_1812688) $ pytest tests/foreman/ui/test_ldap_authentication.py -k test_userlist_with_external_admin 

========================================================= test session starts ==========================================================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.1, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo
plugins: forked-1.1.3, cov-2.8.1, cagoule-0.4.0, mock-1.10.4, xdist-1.34.0, lazy-fixture-0.6.3, services-1.3.1
collecting ... 2020-10-22 11:40:26 - conftest - DEBUG - Collected 36 test cases
collected 36 items / 35 deselected / 1 selected                                                                                        

tests/foreman/ui/test_ldap_authentication.py .                                                                                   [100%]

=========================================================== warnings summary ===========================================================
env/lib/python3.6/site-packages/_pytest/mark/structures.py:337
  /home/okhatavk/Satellite/robottelo/env/lib/python3.6/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.ldap - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

env/lib/python3.6/site-packages/_pytest/mark/structures.py:337
  /home/okhatavk/Satellite/robottelo/env/lib/python3.6/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.high - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======================================== 1 passed, 35 deselected, 2 warnings in 283.19 seconds =========================================


(env) /home/okhatavk/Satellite/robottelo (bz_1812688) $ pytest tests/foreman/ui/test_ldap_authentication.py -k open_ldap
========================================================= test session starts ==========================================================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.1, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo
plugins: forked-1.1.3, cov-2.8.1, cagoule-0.4.0, mock-1.10.4, xdist-1.34.0, lazy-fixture-0.6.3, services-1.3.1
collecting ... 2020-10-22 11:45:48 - conftest - DEBUG - Collected 36 test cases
collected 36 items / 34 deselected / 2 selected                                                                                  
                                                                                                               
tests/foreman/cli/test_ldapauthsource.py .                                                                                                                                             [ 33%]
tests/foreman/ui/test_ldap_authentication.py .s                                                                                                                                        [100%]

============================================================= 1 passed, 1 skipped, 34 deselected, 3 warnings in 250.36 seconds =============================================================
```

**Pull request merged**
